src/NServiceBus.Core/Features/FeatureConfigurationContext.cs

```diff
@@ -37,7 +37,7 @@ public class FeatureConfigurationContext
         /// <summary>
         /// Access to the container to allow for registrations.
         /// </summary>
-        public IConfigureComponents Container { get; }
+        public IServiceCollection Container { get; }
```

Comment:
Same as below, `Container` to `Services`

Reply:
hm do you think that's worth the breaking change?

Reply:
I think it is, it aligns the lingo with the Microsoft one

Reply:
You could also introduce another property and make the current one obselete but forward to the new

Reply:
👆🏼

Reply:
added a `Services` property.